dist/yoast-seo.js

```diff
@@ -264,30 +264,32 @@ YoastSEO.Analyzer.prototype.stopwords = function() {
  * @returns {result object}
  */
 YoastSEO.Analyzer.prototype.fleschReading = function() {
-	var score = (
+	if ( this.preProcessor.__store.wordcountNoTags > 0 ) {
+		var score = (
```

Comment:
Please add the formula for the Flesch reading-ease score so we understand where the numbers are coming from